src/ruby/yast/i18n.rb

```diff
@@ -53,6 +53,12 @@ def _(str)
         end
       end
       FastGettext::Translation::_ str
+    rescue Errno::ENOENT => error
+      if defined?(log) && log.respond_to?(:warn)
```

Comment:
you do not need to depend on log and call `Yast.y2warning` directly https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/logger.rb#L58

just require logger. You do not need to include it.